src/vm.h

```diff
@@ -1,19 +1,25 @@
 #pragma once
 
 #define RAM_SIZE (1 << 30)
-#define KERNEL_OPTS "console=ttyS0"
+#define KERNEL_OPTS "console=ttyS0 pci=conf1"
```

Comment:
You shall explain the kernel option `pci=conf1` within git commit message(s).

Reply:
Without option "pci=conf1", the kernel doesn't find a PCI bus and therefore also doesn't find the virtio-blk devices.